### PR TITLE
NumberField

### PR DESCRIPTION
**Pull request by raggeju**

### CODE REVIEW

File: src/molecules/number-field/NumberField.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+WithDefaultValue.args = {
+	defaultValue: 'Default value',
+};
+
+export const WithPlaceholder = Template.bind({});
+WithPlaceholder.args = {
+	placeholder: 'Input number...',
+};
+
+export const LabelHidden = Template.bind({});
```
Comment (tobiasljungstrom) [inline src/molecules/number-field/NumberField.stories.tsx:50]: This looks redundant

**Review by stt-tomahl — DISMISSED** [review]

File: src/molecules/number-field/NumberField.tsx [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+	border: 1px solid ${Colors.grayDark};
+	cursor: pointer;
+	flex: auto;
+	line-height: 0;
+	padding: 0;
+	position: relative;
+	width: ${Sizes.size_3};
+
+	&:disabled {
+		pointer-events: none;
```
Comment (tobiasljungstrom) [inline src/molecules/number-field/NumberField.tsx:159]: I think the buttons should have `cursor: not-allowed;` when disabled.

**Review by tobiasljungstrom — COMMENTED** [review]
Good stuff!

File: src/molecules/number-field/NumberField.stories.tsx (hunk 1 above)
Comment (raggeju) [reply src/molecules/number-field/NumberField.stories.tsx:50]: Solved it by adding { label: '' }

**Review by tobiasljungstrom — APPROVED** [review]

**Review by stt-tomahl — APPROVED** [review]

**Pull request merged**